src/ui/components/chart/Chart.tsx

```diff
@@ -142,12 +146,18 @@ function getSegmentColor({
   });
 }
 
-export function Chart({
+export function Chart<T>({
   chartPoints,
   onRangeSelect,
   style,
+  datasetConfig = {},
+  tooltip = {},
+  plugins = [],
+  interaction = {},
```

Comment:
I actually do not know whether using default values in useEffect dependencies will cause updates 🤔 Can you check this please?